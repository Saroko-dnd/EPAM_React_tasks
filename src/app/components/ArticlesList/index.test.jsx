import ArticleList from '.';

describe('render an ArticleList', () => {
    const listOfArticles = shallow(<ArticleList articles={[]} />);

    it('should render ArticleList correctly', () => {
        expect(listOfArticles).toMatchSnapshot();
    });
});
