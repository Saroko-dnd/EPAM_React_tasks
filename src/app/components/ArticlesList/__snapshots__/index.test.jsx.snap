// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render an ArticleList should render ArticleList correctly 1`] = `
<div
  className="news-list "
>
  <h2
    className="title"
  />
  <div
    className="d-flex justify-content-center align-items-center flex-wrap justify-content-center"
  />
</div>
`;
